fe/fe-core/src/main/java/com/starrocks/leader/LeaderImpl.java

```diff
@@ -1219,7 +1220,8 @@ public TBeginRemoteTxnResponse beginRemoteTxn(TBeginRemoteTxnRequest request) th
             txnId = GlobalStateMgr.getCurrentState().getGlobalTransactionMgr().beginTransaction(db.getId(),
                     request.getTable_ids(), request.getLabel(),
                     new TxnCoordinator(TxnSourceType.FE, FrontendOptions.getLocalHostAddress()),
-                    LoadJobSourceType.valueOf(request.getSource_type()), request.getTimeout_second());
+                    LoadJobSourceType.valueOf(request.getSource_type()), request.getTimeout_second(),
+                    WarehouseManager.DEFAULT_WAREHOUSE_ID);
```

Comment:
‌‌‌‌There seems to be an issue with this WarehouseID. 

By the way, could you provide some context on how beginRemoteTxn is being called?

Reply:
This is just the mobile logic, maintaining the logic in ee